src/components/Accordion/docs/AccordionPropsTable.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+import { PropsTable } from '../../../docs/PropsTable';
+
+export const AccordionPropsTable = () => {
+    const props = [
```

Comment:
If this is a static array it could be defined outside to not recreate the array on each call